### PR TITLE
docs: Add link to mst-query

### PR DESCRIPTION
**Pull request by k-ode**
@jamonholmgren suggested I add a link to mst-query (a query library for mobx-state-tree) to the docs. I didn't see any lists for addons or libraries so I added one in the resources file.

### CODE REVIEW

File: docs/tips/resources.md [hunk 1]
```diff
@@ -37,3 +37,7 @@ title: Talks & Blogs
 - [Reactotron](https://github.com/infinitered/reactotron)
 - [MobX DevTools](https://chrome.google.com/webstore/detail/mobx-developer-tools/pfgnfdagidkfgccljigdamigbcnndkod)
 - The Redux can be connected as well as demonstrated [here](https://github.com/mobxjs/mobx-state-tree/blob/1906a394906d2e8f2cc1c778e1e3228307c1b112/packages/mst-example-redux-todomvc/src/index.js#L6)
+
+## Addons, libraries and tools
+
+- [mst-query](https://github.com/ConrabOpto/mst-query/)
```
Comment (jamonholmgren) [inline docs/tips/resources.md:43]: Let's add some context!

```suggestion
- [mst-query](https://github.com/ConrabOpto/mst-query/) - Query library for MST with support for auto normalization, garbage collection, infinite scroll & pagination, and more
```

**Review by jamonholmgren — CHANGES_REQUESTED** [review]
Feel free to update my wording, but I made a stab at it.

**Comment by k-ode** [comment]
@jamonholmgren Updated, thanks!

**Review by jamonholmgren — APPROVED** [review]

**Comment by jamonholmgren** [comment]
Thank you @k-ode !

**Pull request merged**